### PR TITLE
Enhance readme

### PR DESCRIPTION
**Pull request by lukesleeman**
This updates the readme to make it easier for new members to join

## Changes

- Added app information architecture and plan
- Explained dev process
- Updated authors
- Added emergency contacts

## Things to note

I'm going to get @ericjiang97 to fill out the flutter specific stuff in a new PR once this gets merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,49 @@
...
+⬇️
+ 
+🖥 **CI** builds the app and deploys to phones via firebase app distribution.
+
+
+## The Plan
+
+The app will have these screens and sections:
+
+![App information architecture](docs/Flow-Diagram.png "The information architecture of the app")
```
Comment (ericjiang97) [inline README.md]: Image wouldn't load.

```
![App information architecture](docs/Flow-Diagram.png)
```

Comment (ericjiang97) [reply README.md]: oh never mind.

**Comment by lukesleeman** [comment]
I've finished all the tweaks and polishing on this now.  Should be good to get some 👀 on it

File: README.md [hunk 2]
```diff
@@ -1,17 +1,49 @@
-# EGBA
+# Western Health COVID-19 app
 
-> Everythings
+A tool to safeguard the welfare of Western Health anaesthetists during COVID19 and provide instant access to the most up-to-date information for time-critical procedures to ensure optimal patient care.  This app will contain information for frontline medical staff on subjects such as:
```
Comment (kimngyn) [inline README.md]: COVID-19 for consistency

File: README.md [hunk 3]
```diff
@@ -1,17 +1,49 @@
...
+- Ventilation suggested initial actions and adjuncts
+- ICU General Care
+- ICU Tips for junior staffers
+- Contact details and newsfeeds
 
-> Be
 
-> Alright
-
-A tool to safeguard the welfare of Western Health anaesthetists during COVID19 and provide instant access to the most up-to-date information for time-critical procedures to ensure optimal patient care.  Currently under development by a community of volunteers.
```
Comment (kimngyn) [inline README.md:11]: COVID-19

File: README.md [hunk 4]
```diff
@@ -37,14 +69,26 @@ Designs are hosted in zeplin cloud.  DM `@luke.sleeman` on GDG Melbourne slack t
...
 
-Gregg Miller - Initial Development and ideas
-
-Matt Kelsh - Design
+* Gregg Miller - Initial Development and ideas
+* Matt Kelsh - Design
+* Marc Edwads - Design and management
+* Luke Sleeman - Organising people and general dogsbody 😁
+* Alessandro Favero - Flutter Dev
+* Makim Lin - Flutter dev & CI
```
Comment (kimngyn) [inline README.md]: Maksim

File: README.md [hunk 5]
```diff
@@ -37,14 +69,26 @@ Designs are hosted in zeplin cloud.  DM `@luke.sleeman` on GDG Melbourne slack t
...
+* Mike Huges - Flutter dev & CI
```
Comment (kimngyn) [inline README.md]: Hughes

File: README.md [hunk 6]
```diff
@@ -37,14 +69,26 @@ Designs are hosted in zeplin cloud.  DM `@luke.sleeman` on GDG Melbourne slack t
...
+* Adam Koch - Flutter dev
+* Eric Jiang - Flutter dev, and QA
 
 ## License
 
 This project is licensed under the Apache 2.0 - see the `LICENSE-2.0.txt` file for details.
 
+## Emergency contacts
+
+For any urgent requests relating to this app you can contact luke.sleeman@gmail.com - It will go to Lukes phone.  You can also DM luke through [GDG Melbourne's slack](http://bit.ly/join_gdgslack) - DM `@luke.sleeman`.
```
Comment (kimngyn) [inline README.md]: Luke, capital L

**Review by lukesleeman — COMMENTED** [review]
.

File: README.md [hunk 7]
```diff
@@ -37,14 +69,26 @@ Designs are hosted in zeplin cloud.  DM `@luke.sleeman` on GDG Melbourne slack t
...
+
 ## Acknowledgments
 
-🎩 Huge hat tip to everybody providing advice, helping out, etc including: Gregg Miller, Matt Kelsh, Mike Hughes,  Alessandro Favero, Maksim Lin, Brett Morgan, 
\ No newline at end of file
+🎩 Huge hat tip to everybody providing advice, helping out, etc including: Gregg Miller, Matt Kelsh, Mike Hughes,  Alessandro Favero, Maksim Lin, Brett Morgan, Adam Koch, Eric Jiang, Hsiu-Mei Huang, Sean Woodhouse, Matt Delves.
+
+I've probably forgotten a bunch of people, so if you have done something and we have missed out on you;  I'm deeply sorry, and appreciate your contribution ❤️.  Message Luke Sleeman and he will make sure you get added.
```
Comment (kimngyn) [inline README.md:95]: Well Done! Great Job Luke

File: README.md (hunk 4 above)
Comment (kimngyn) [reply README.md]: and Edwards

File: README.md (hunk 6 above)
Comment (kimngyn) [reply README.md]: Luke's

File: README.md (hunk 2 above)
Comment (lukesleeman) [reply README.md]: Done 👍

File: README.md (hunk 4 above)
Comment (lukesleeman) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (lukesleeman) [reply README.md]: Good catch, fixed!

File: README.md (hunk 6 above)
Comment (lukesleeman) [reply README.md]: Well spotted, fixed!

File: README.md (hunk 7 above)
Comment (lukesleeman) [reply README.md:95]: No worries, thanks for your help 😁

**Review by maks — APPROVED** [review]
👍

File: README.md [hunk 8]
```diff
@@ -37,14 +69,27 @@ Designs are hosted in zeplin cloud.  DM `@luke.sleeman` on GDG Melbourne slack t
...
-
-Matt Kelsh - Design
+* Gregg Miller - Initial Development and ideas
+* Matt Kelsh - Design
+* Marc Edwards - Design and management
+* Luke Sleeman - Organising people and general dogsbody 😁
+* Alessandro Favero - Flutter Dev
+* Maksim Lin - Flutter dev & CI
+* Mike Hughes - Flutter dev & CI
+* Brett Morgan
```
Comment (domesticmouse) [inline README.md:79]: Adam Koch should go before me in this list. Maybe sort alphabetically?

**Review by domesticmouse — APPROVED** [review]

**Pull request merged**